src/style.css

```diff
@@ -0,0 +1,99 @@
+/* #region fonts */
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 700;
+}
+
+/* #endregion fonts */
+
+/* #region base-styles */
+input {
```

Comment:
It is better to tie stylizations more to classes and work directly with tags as little as possible.